content/_ext/yaml_gallery_generator.py

```diff
@@ -60,6 +60,7 @@ def _generate_menu(all_items, flt=None):
         menu_html += _generate_tag_menu(all_items, tag_key) + '\n'
     if flt:
         menu_html += '<a type="button" class="btn btn-link" href="/pages/links.html">Return to Full Gallery</a>' + '\n'
+    menu_html += '<a type="button" class="btn btn-link" style="position:absolute; right:0;" href="test.html">Submit a Link</a>' + '\n'
```

Comment:
@jukent, once #86 is merged, the `href` here should point to this URL: `https://github.com/ProjectPythia/projectpythia.github.io/issues/new?assignees=&labels=external-links-gallery-submission&template=update-external-links-gallery.md&title=`

This will take the user to the `open new issue` page with the right template.

Reply:
Done! Thanks :)